composer.json

```diff
@@ -25,15 +25,23 @@
         "cweagans/composer-patches": "1.6.7",
         "drupal/core-recommended": "8.8.4",
         "drush/drush": "9.7.1",
-        "drupal/background_image_formatter": "1.4",
         "drupal/bootstrap_barrio": "4.22",
+        "drupal/cas": "1.6.0",
+        "drupal/externalauth": "1.2.0",
+        "drupal/background_image_formatter": "1.4",
```

Comment:
Nitpick: I'd be preferable to keep these in alphabetical order. We could add this to our composer.json (like we have in our scaffolding repo) to make these easier to do automatically:

"config": {
    "sort-packages": true
},